lessons/lesson4.md

```diff
@@ -10,109 +10,72 @@ There are three ways one can consume event:
 For enterprise developers, Adobe offers journaling to consume events. The Adobe I/O Events Journaling API enables enterprise integrations to consume events according to their own cadence and process them in bulk. Unlike webhooks, no additional registration or other configuration is required; every enterprise integration that is registered for events is automatically enabled for journaling. Journaling data is retained for 7 days. 
 
 After you fire event, you should be able to verify your event through journaling `UNIQUE API ENDPOINT` you get from console by follow below instruction
-[Journaling api](https://www.adobe.io/apis/experienceplatform/events/docs.html#!adobedocs/adobeio-events/master/intro/journaling_api.md)
+[Journaling api](https://www.adobe.io/apis/experienceplatform/events/docs.html#!adobedocs/adobeio-events/master/api/journaling_api.md)
 you could use `Curl` command or `POSTMAN` to call this journaling `UNIQUE API ENDPOINT` to see your fired event.
 Or you can use [Custom event SDK](https://github.com/adobe/aio-lib-events/) to call Journaling API to retrieve your event.
 
 
 ### Option 2: Using runtime action
 Once you have access to [Adobe I/O Runtime](https://www.adobe.io/apis/cloudplatform/runtime.html) (in our case you already have) and you have your [slack webhook url defined](https://api.slack.com/incoming-webhooks) :
 * Edit the `manifest.yml` to add an action called slack 
+Note: below are the sample code for example, please modify to your own need 
 ```javascript
- slack:
```

Comment:
was the manifest example accidentally deleted?

Reply:
aha....bumper... good eye, fix it